core/adminserver/admin_server_test.go

```diff
@@ -101,19 +99,36 @@ func TestCreateDomain(t *testing.T) {
 			expect:   func(e *miniEnv) {},
 		},
 		{
-			desc:     "Map init fails",
+			desc:     "Create map fails",
 			domainID: "mapinitfails",
 			wantCode: codes.Internal,
 			expect: func(e *miniEnv) {
 				e.s.Admin.EXPECT().CreateTree(gomock.Any(), gomock.Any()).Return(&tpb.Tree{TreeType: tpb.TreeType_LOG}, nil)
 				e.s.Log.EXPECT().InitLog(gomock.Any(), gomock.Any()).Return(&tpb.InitLogResponse{}, nil)
 				e.s.Log.EXPECT().GetLatestSignedLogRoot(gomock.Any(), gomock.Any()).Return(&tpb.GetLatestSignedLogRootResponse{}, nil)
 				e.s.Admin.EXPECT().CreateTree(gomock.Any(), gomock.Any()).Return(&tpb.Tree{TreeType: tpb.TreeType_MAP}, nil)
-				e.s.Map.EXPECT().InitMap(gomock.Any(), gomock.Any()).Return(&tpb.InitMapResponse{}, fmt.Errorf("init map failure")).MinTimes(1)
+				e.s.Map.EXPECT().InitMap(gomock.Any(), gomock.Any()).Return(&tpb.InitMapResponse{}, nil)
+				e.s.Map.EXPECT().GetSignedMapRootByRevision(gomock.Any(), gomock.Any()).Return(&tpb.GetSignedMapRootResponse{}, fmt.Errorf("not found")).MinTimes(1)
```

Comment:
The indentation looks screwy here but it's probably Github's fault, I can't tell.